packages/blade/src/components/Box/BaseBox/types/spacingTypes.ts

```diff
@@ -1,14 +1,13 @@
 import type { MakeObjectResponsive } from './responsiveTypes';
 import type { DotNotationSpacingStringToken } from '~src/_helpers/types';
-import type { Platform } from '~utils';
 
 type ArrayOfMaxLength4<T> = readonly [T?, T?, T?, T?];
-type SpaceUnits = Platform.Select<{
-  web: 'px' | '%' | 'fr' | 'rem' | 'em' | 'vh' | 'vw';
-  native: 'px' | '%';
-}>;
-type SpacingValueType = DotNotationSpacingStringToken | `${string}${SpaceUnits}` | 'auto';
-
+type SpaceUnits = 'px' | '%' | 'fr' | 'rem' | 'em' | 'vh' | 'vw';
```

Comment:
Platform.Select is not working as we expect here. I think someone had also raised an issue about this earlier on slack. For now I've made all these values available on both platforms. If someone uses an unsupported value on RN, RN will throw big red errors

Reply:
I was thinking perhaps we can just make this `string`, don't know if the typing is adding much value

Reply:
So the issue that was raised should be unrelated to that. The types are broken because of forwardRef. Have found a fix for that, will create PR 🤔

Reply:
#1272